README.md

```diff
@@ -8,18 +8,18 @@
 
 TPI is a [Terraform](https://terraform.io) plugin built with machine learning in mind. Full lifecycle management of computing resources (including GPUs and respawning spot instances) from several cloud vendors (AWS, Azure, GCP, K8s)... without needing to be a cloud expert.
 
-- **Provision Resources**: create cloud compute (CPU, GPU, RAM) & storage resources without reading pages of documentation
-- **Sync & Execute**: easily sync & run local data & code in the cloud
-- **Low cost**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
+- **Lower cost with spot recovery**: transparent auto-recovery from interrupted low-cost spot/preemptible instances
```

Comment:
made a lot of changes since this review, a couple outstanding things:

> We are not stoping tasks, we are destroying tasks

technically we *are* stopping tasks <- terminating instances <- destroying resources

> automatic intermediate data backup

implied by transparent spot auto-recovery, right?

Reply:
> implied by transparent spot auto-recovery, right?

Yes and no, you make the users to think or guess at least. There is a phase in design (title also of a praised book) "Dont make me think"

Reply:
> technically we are stopping tasks

No, technically Tasks are stopped by themselves, what I want to say with this is that the user might find the ambiguity of the stop concept in cloud. Where they can stop the machine and then at some point they can restart the machine again. Here after destroy there is no possible resume